### PR TITLE
Merge branch delete course command 15 oct into master

### PR DESCRIPTION
**Pull request by RCPilot1604**

### CODE REVIEW

File: src/main/java/tutorlink/commandpackage/AddCourseCommand.java [hunk 1]
```diff
@@ -30,7 +31,7 @@ public CommandResult execute() {
             Course course = new Course(this.courseID, this.courseMCs);
             CourseList courseList = students.getStudent(this.matricNumber).courses;
             courseList.addCourse(course);
-            return new CommandResult(String.format(SUCCESSFUL_MESSAGE,course));
+            return new CommandResult(String.format(SUCCESSFUL_MESSAGE,course,this.matricNumber));
```
Comment (yeekian) [inline src/main/java/tutorlink/commandpackage/AddCourseCommand.java:34]: I like that you incorporated the matricNumber in the print string as it allows us to display more information

**Review by yeekian — APPROVED** [review]
LGTM!

File: src/main/java/tutorlink/commandpackage/DeleteCourseCommand.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        this.courseID = courseID;
+        this.matricNumber = matricNumber;
+    }
+
+    @Override
+    public CommandResult execute() {
+        try{
+            CourseList courseList = students.getStudent(this.matricNumber).courses;
+            courseList.dropCourse(this.courseID);
+            return new CommandResult(String.format(SUCCESS_MESSAGE,this.courseID,this.matricNumber));
```
Comment (jinzihan2002) [inline src/main/java/tutorlink/commandpackage/DeleteCourseCommand.java:27]: I like that DeleteCourse checks for the existence of the course to be deleted, so the exception can be handled.

**Review by jinzihan2002 — COMMENTED** [review]
Overall LGTM

**Pull request merged**